packages/zeebe-element-templates-json-schema/src/defs/properties/taskSchedule.json

```diff
@@ -0,0 +1,88 @@
+{
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "binding": {
+            "properties": {
+              "type": {
+                "const": "zeebe:taskSchedule"
+              },
+              "property": {
+                "enum": [
+                  "dueDate",
+                  "followUpDate"
+                ]
+              }
+            },
+            "required": [
+              "type",
+              "property"
+            ]
+          }
+        },
+        "required": [
+          "binding"
+        ]
+      },
+      "then": {
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "enum": [
+              "Hidden",
+              "String",
+              "Dropdown",
+              "Text"
+            ]
+          }
+        }
+      }
+    },
+    {
+      "if": {
+        "properties": {
+          "binding": {
+            "properties": {
+              "type": {
+                "const": "zeebe:taskSchedule"
+              },
+              "property": {
+                "enum": [
+                  "dueDate",
+                  "followUpDate"
+                ]
+              }
+            },
+            "required": [
+              "type",
+              "property"
+            ]
+          }
+
+        },
+        "required": [
+          "binding",
+          "value"
+        ],
+        "not": {
+          "required": ["feel"]
+        }
+      },
+      "then": {
+        "properties": {
+          "value": {
+            "type": "string",
+            "pattern": "^(?<date>\\d{4}-(?<month>0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01]))T(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9](Z|([+-](0[0-9]|1[0-3]):[0-5][0-9](\\[[^\\]]+\\])?))$",
```

Comment:
[We decided](https://github.com/camunda/camunda-modeler/issues/5093#issuecomment-2983009652) to utilize the regex also used by the linting infrastructure. This is the inlined version of that. I am also fine with dropping this feature if we do deem it too unmaintainable. However, I do think it provides user value by ensuring the template uses correct dates.